docs/_themes/clusterhq/layout_docs.html

```diff
@@ -65,6 +65,8 @@
             })();
         </script>
     {% endif %}
+	
+    <link rel="shortcut icon" href="https://clusterhq.com/assets/favicon.ico"/>
```

Comment:
Hm.  Considering clusterhq.com is maintained separately from this, do we want this foreign dependency?  What does @msh100 think about docs.clusterhq.com reaching over and grabbing clusterhq.com's favicon.ico?